Angular template for config app.

emanuelschuetze opened the pull request:
Ready for first review.

My Iast TODOs:
* updated config (requires #1529)
* reset all configs to default values

@normanjaeckel I have some suggestions for you to update the rest-api:
* use new prefix "general_" for all general config keys
* introduce new "type" value for each config field to show the correct html input type in list.html (e.g. radio, checkbox, text, number etc.; see http://www.w3schools.com/html/html_form_input_types.asp)
* rename config names: "motion" -> "motions"; "assignment" -> "assignments"

emanuelschuetze commented:
@normanjaeckel A "label" and a optional "help" field would be nice for each config field to show config forms like in OpenSlides 1.x

ostcar commented:
@emanuelschuetze to fix travis, you only have to wait for #1539 and rebase on top of it (which should be easy, because you did not touch so much python code)

normanjaeckel commented:
See #1545 for new config var names. New type is still missing.

emanuelschuetze commented:
Rebased. I'll now add the update function.
@normanjaeckel Please add a issue/pull request with your last TODO (add new type).

ostcar commented on openslides/urls.py:
```diff
@@ -11,6 +11,7 @@
     url(r'^', include('openslides.core.urls')),
     url(r'^agenda/', include('openslides.agenda.urls')),
     url(r'^assignments/', include('openslides.assignments.urls')),
+    url(r'^config.*', IndexView.as_view()),
```
this line can be removed, because we have a fallback to to IndexView now.

normanjaeckel commented:
Ok.

emanuelschuetze commented:
Ready for review.

ostcar commented:
@emanuelschuetze one note, then it can be merged

ostcar commented:
This pull request should be rebased ontop of #1549 because it will break after this pull request is merged.

emanuelschuetze commented:
Fix one remark. Rebased. I'll wait until #1549 is merged to update and rebase.

emanuelschuetze commented:
@ostcar I'll wait for your pullrequest for #1558.

emanuelschuetze commented:
Rebased and added code to reset to default_value.
Still open: We have to check if the input is in right input type _before_ sending PUT request.. Otherwise the PUT-Request failed and shows some more errors.

normanjaeckel commented:
@emanuelschuetze: You may want this to be merged and move the last hint to an extra issue?

emanuelschuetze commented:
If @ostcar can do the last pull request (or give me some hints to do that) in the next days I will wait for it and merge it together... Currently my solution does not work for e.g. config choices.

ostcar commented:
@emanuelschuetze it would be easier for me if you rebased and merged this (unfinished) pull request. Then I can fix the config field on top of it.

ostcar commented:
@emanuelschuetze Please move the code in config/static to core/static

emanuelschuetze commented:
Moved code from config/static to core/static. Rebased.
Ready for review and merge.
